Makefile

```diff
@@ -11,7 +11,7 @@ SHELL := /bin/bash
 # Some handy utilities
 GLOB = git ls-files -z $1 | tr '\0' '\n' | xargs -I {} find {} ! -type l
 NPM_INSTALL = npm prune --production=false && npm install
-BOWER_INSTALL = bower install --config.registry.search=http://registry.origami.ft.com --config.registry.search=https://bower.herokuapp.com
+BOWER_INSTALL = bower prune && bower install --config.registry.search=http://registry.origami.ft.com --config.registry.search=https://bower.herokuapp.com
```

Comment:
I renamed .bowerrc to .bowerrcccc locally. bower prune succeeded, but then bower install failed, so it looks like prune is a wholly local process